### PR TITLE
Adding Payee/Description field to QIF export.

### PR DESCRIPTION
**Pull request by MisterY**
Adding Payee/Description field to QIF export and moving the Memo to Memo QIF field.
Implements #343 as discussed there.

### CODE REVIEW

**Comment by rivaldi8** [comment]
Merged manually onto hotfix/patches (26277c6). Thanks!

File: app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java [hunk 1]
```diff
@@ -144,4 +149,46 @@ public void multiCurrencyTransactions_shouldResultInMultipleQifFiles(){
         assertThat(file.length()).isGreaterThan(0L);
     }
 
+    //@Test
+    public void description_and_memo_field_test() {
```
Comment (rivaldi8) [inline app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java]: Please, follow Java naming conventions next time. `testDescriptionAndMemoField()` would've been better.

Comment (MisterY) [inline app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java]: Alright, thanks for the suggestion. I do, however, disagree that camel-case is an appropriate syntax for long test names. It is completely unreadable for anything longer than three words.
Even the name of the existing test combines the two standards, which is even worse, in my opinion.

Comment (rivaldi8) [inline app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java]: Yes, I also disagree with some conventions, but not following them just makes the code stand out unnecessarily.

The tests combining the two styles is also a convention :-) I can't find the reference now, but more or less it follows the format: `someAction_whatDoYouExpectFromIt()`.

**Pull request closed without merging**